lib/util/state.js

```diff
@@ -255,7 +259,8 @@ export const getActiveItineraries = createSelector(
     const {direction, type} = sort
     // If no sort type is provided (e.g., because batch routing is not enabled),
     // do not sort itineraries (default sort from API response is used).
-    return !type
+    // Also, do not sort itineraries if the a field trip request is active
+    return (!type || Boolean(activeFieldTripRequest))
```

Comment:
I think type is set to null unless batch routing is enabled. Batch routing and the field trip shouldn't generally be enabled at the same time. This change is probably ok, but I don't necessarily think it's needed.